README.md

```diff
@@ -292,12 +292,27 @@ v0.0.3
 
 ```
 # Helper Functions
+## cap_array_value
+Retrieves a value from an array file based on a zero based index.
+```
+cap_arrary_value FILE [INDEX]
+```
+- `FILE` The file containing an array value on each line.
+- `INDEX` The optional zero based index for the value of the array. If a value
+    is not provided then SLURM_ARRAY_TASK_ID environment variable will be used
+    as the default.
```

Comment:
I am unsure if I am using `INDEX` correctly. I modified the testing script by setting `INDEX` to 1 to see what happens, but when I do that, the same "sample" (aka person's name, in this case "Lizzy") gets used for each array job. Is this how it's intended to function?

Reply:
yes, I think that's how it works

Reply:
I discussed this with Tonie offline and got clarification.